sdk/tests/conformance2/transform_feedback/default_transform_feedback.html

```diff
@@ -0,0 +1,142 @@
+in float in_value;
+out float out_value;
+
+void main() {
+   out_value = in_value * 2.;
+}
+</script>
+<script id="fshader" type="x-shader/x-fragment">#version 300 es
+precision mediump float;
+out vec4 dummy;
+void main() {
+  dummy = vec4(1);
+}
+</script>
+<script>
+"use strict";
+description("This test verifies using the default transform feedback object");
+
+debug("");
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+var gl = wtu.create3DContext(canvas, null, 2);
+
+if (!gl) {
+    testFailed("WebGL context does not exist");
+} else {
+    testPassed("WebGL context exists");
+
+    runDefaultTransformFeedbackTest();
+    finishTest();
+}
+
+function runDefaultTransformFeedbackTest() {
+  const prog = wtu.setupTransformFeedbackProgram(gl, ["vshader", "fshader"],
+      ["out_value"], gl.SEPARATE_ATTRIBS,
+      ["in_value"]);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "linking transform feedback shader should not set an error");
+
+  const inLoc = 0;
+  const outLoc = 0;
+
+  const srcBuffer1 = createBuffer(gl, new Float32Array([1, 2, 3]));
+  const srcVAO1 = createVAO(gl, srcBuffer1, inLoc);
+
+  const dstBuffer = createBuffer(gl, Float32Array.BYTES_PER_ELEMENT * 3);
+
+  const tf = null; // use the default transform feedback gl.createTransformFeedback();
+  gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+  gl.useProgram(prog);
+  gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, dstBuffer);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
+
+  runFeedback(gl, prog, srcVAO1, tf);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
+
+  const expected = [2, 4, 6];
+  gl.bindBuffer(gl.ARRAY_BUFFER, dstBuffer);
+  wtu.checkFloatBuffer(gl, gl.ARRAY_BUFFER, expected);
+}
+
+function runFeedback(gl, prog, srcVAO, tf, dstBufferInfo) {
+  gl.enable(gl.RASTERIZER_DISCARD);
+
+  gl.useProgram(prog);
+  gl.bindVertexArray(srcVAO);
+
+  gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+  gl.beginTransformFeedback(gl.TRIANGLES);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
+  gl.drawArrays(gl.TRIANGLES, 0, 3);
+  gl.endTransformFeedback();
+
+  // FIXME: this should not be needed once the spec and implementationsa are fixed
+  gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
+
+  gl.disable(gl.RASTERIZER_DISCARD);
+}
+
+function createBuffer(gl, dataOrSize) {
+  const buf = gl.createBuffer();
+  gl.bindBuffer(gl.ARRAY_BUFFER, buf);
+  gl.bufferData(gl.ARRAY_BUFFER, dataOrSize, gl.STATIC_DRAW);
+  return buf;
+}
+
+function createVAO(gl, buf, inLoc) {
+  const vao = gl.createVertexArray();
+  gl.bindVertexArray(vao);
+  gl.bindBuffer(gl.ARRAY_BUFFER, buf);
+  gl.enableVertexAttribArray(inLoc);
+  gl.vertexAttribPointer(inLoc, 1, gl.FLOAT, false, 0, 0);
+  // FIXME: this should not be needed once the spec and implementationsa are fixed
+  gl.bindBuffer(gl.ARRAY_BUFFER, null);  // this is not needed
```

Comment:
This is unnecessary even for now - srcBuffer1 is never bound to TF, so there is no conflict at all, no need to unbind it. Therefore, this FIXME is misleading.